tests/Mock/Connection/TestBroker.php

```diff
@@ -21,6 +21,8 @@ class TestBroker extends RestClient
 
     protected $apiEndpoint;
 
+    protected $responseData = '{"message": "Some JSON Response Data", "id": "1234"}';
```

Comment:
Personally, if you are not going to reuse this property on other methods, there is no need to define it as shared/protected property, and its content can be moved directly to line 57, inside the `json_decode` function.